### DISCUSSION TITLE
Don't show Firefox popup bar on top of TST

### DISCUSSION THREAD

**Discussion by sarmong** [piroor/treestyletab#3404] [Q&A]
On some websites Firefox shows a popup bar with some info that shows up right below the bookmarks bar:
![image](https://github.com/piroor/treestyletab/assets/42828375/e56bde11-62af-495f-8797-78d9a5205a95)
![image](https://github.com/piroor/treestyletab/assets/42828375/b64585d1-11e5-4873-a857-63ca1a15b80a)

My problem is that it also shows above the TST. When I switch tabs, TST jumps up and down making it harder to navigate.

Can I style this bar to make it narrower and appear only on top of the site content? 
Do you think it makes sense to add it into the extension code?

**Comment by irvinm** [comment]
This is not TST related nor even sidebar related.  There is nothing TST could do to change this behavior.

However, having said all of that, it might be possible to hide (or change) such elements using userChrome.css.  (https://superuser.com/questions/1608096/how-to-inspect-firefoxs-ui)  It isn't super difficult, but it isn't easy either.

The other choice might be to ask the same question over at https://www.reddit.com/r/FirefoxCSS/ to help to hide those notices.
